File: JsonInflatorIOS/JsonInflator/Classes/utility/InflatorDimensionLookup.swift
//
//  InflatorDimensionLookup.swift
//  Json inflator Pod
//
//  Library utility: protocol for dimension lookup
//  Integrates with InflatorConvUtil to look up custom dimension sizes or coordinates
//

public protocol InflatorDimensionLookup {
    
    func getDimension(refId: String) -> CGFloat?
    
}

File: JsonInflatorAndroid/JsonInflatorLib/src/main/java/com/crescentflare/jsoninflator/utility/InflatorDimensionLookup.java
package com.crescentflare.jsoninflator.utility;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

/**
 * Json inflator utility: interface for dimension lookup
 * Integrates with InflatorMapUtil to look up custom dimension references
 */
@SuppressWarnings({"unused", "WeakerAccess"})
public interface InflatorDimensionLookup
{
    @Nullable
    Integer getDimension(@NotNull String refId);
}
